### PR TITLE
fix: suggest phrasing improvements

### PR DESCRIPTION
**Pull request by eliotlim**
+ suggest changes to phrasing
+ suggest modifications to spelling

### CODE REVIEW

File: doc/l3-competency-writeup.md [hunk 1]
```diff
@@ -5,23 +5,23 @@ This write up is more applicable for Backend Engineering competency, but a simil
 
 ```text
```
Comment (abeltay) [inline doc/l3-competency-writeup.md:6]: offtopic: What do you think about using a normal header to allow the text below to be non-monospaced and wrapped. Seems easier to read with that formatting.

Comment (eliotlim) [reply doc/l3-competency-writeup.md:6]: Hmm it's possible! I'm open to it :)

I think @cxcheng may have intended this section to act as a reference, and (to me) using the code block intuitively engages that part of a SWE's mind? 😅

Comment (cxcheng) [reply doc/l3-competency-writeup.md:6]: I did that to be aligned with the form.gov.sg input field which is in plain text

Comment (eliotlim) [reply doc/l3-competency-writeup.md:6]: Okay, it sounds like there's a rationale behind it! If you'd like to see what such a change would look like, you could always open another PR 😄 

I'll mark this conversation as resolved, then. ✅
Thanks for your input!

**Review by abeltay — APPROVED** [review]

**Pull request merged**